tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/worker/BasicWorkerTest.java

```diff
@@ -109,6 +109,19 @@ public void testSimpleSyncSendFunctions(String funcName) {
         };
     }
 
+    @Test(dataProvider = "basicWorkerFunctions")
```

Comment:
I'm not sure if this is the correct place for these tests given that we are testing something that is impacted by an implementation decision.  Also, just looking at the test names, it is not clear what we are testing.